docs/cookbook/deploy-using-bentoml.md

```diff
@@ -0,0 +1,201 @@
+# Run Outlines using BentoML
+
+[BentoML](https://bentoml.com/) is an open-source model serving library for building performant and scalable AI applications with Python. It comes with tools that you need for serving optimization, model packaging, and production deployment.
+
+In this guide, we will show you how you can use BentoML to run programs written with Outlines on GPU locally and in the cloud. The example source codes in this guide are also available at [examples/bentoml/](https://github.com/outlines-dev/outlines/blob/main/examples/bentoml/) directory.
+
+## Import model
+
+First we need to download the Mistral-7B-v0.1 model and import the model into BentoML's model store. Let's install the BentoML and other dependencies from PyPi (preferably in a virtual environment):
+
+```bash
+pip install -r requirements.txt
+```
+
+Then save the code snippet below as `import_model.py` and run `python import_model.py`
```

Comment:
I tried and it looks like users need to gain access first or the download will fail.


```suggestion
Then save the code snippet below as `import_model.py` and run `python import_model.py`.

**Note**: You need to accept related conditions on [Hugging Face](https://huggingface.co/mistralai/Mistral-7B-v0.1) first to gain access to Mistral-7B-v0.1.
```